src/ontodia/workspace/workspace.ts

```diff
@@ -127,6 +132,10 @@ export class Workspace extends Component<WorkspaceProps, State> {
         } as MarkupProps & React.ClassAttributes<WorkspaceMarkup>);
     }
 
+    protected getFlowLayoutHandler = (): (() => void) => {
```

Comment:
Use a usual function definition here, e.g.:
```
protected getFlowLayoutHandler(): () => void {
  // ...
}
```

Reply:
In that case we wont be able to use 'this' in overridden function, or we will need to bind this function. Is it OK?